flowmachine/flowmachine/features/subscriber/displacement.py

```diff
@@ -42,11 +42,11 @@ class Displacement(SubscriberFeature):
         e.g. 2016-01-01 or 2016-01-01 14:03:01
     stop : str
         As above
-    modal_locations : ModalLocation
+    reference_location : ModalLocation
         The set of home locations from which to calculate displacement.
         If not given then ModalLocation Query wil be created over period
         start -> stop.
-    statistic : str
+    value : str
```

Comment:
Actually, the _input_ parameter should still be called `statistic` (otherwise it's not really intuitive for the user what it means). So this needs to be changed back.

For clarity of anyone reading the code I would also suggest to keep the name `statistic` anywhere internally in this class.

So I'd only change the column name of the returned output to the name `value` (as you have done in L193 below).